e2e/utils_test.go

```diff
@@ -346,3 +348,26 @@ func loopDevicesCleanup(t *testing.T) error {
 func formatVolName(volName string) string {
 	return strings.Replace(volName, "/", "-", 1)
 }
+
+func IsDaemonRunning(pidpath string) bool {
```

Comment:
As all tests that use this function are present in the same `e2e` package, please do not export this function. 
Suggestion: You can rename this function as `isProcessRunning` to be slightly broader. In future, tests may use this to check if any process is running, not necessarily those processes that turn into daemons.  

Clean implementation though :+1:

Reply:
done